src/parser/asts/functions.c

```
@@ -1,215 +1,168 @@
-	node->left->left = params;
-
-	index = params->endingIndex + 2;
-
-	int buffSize = 32;
-	int buffIndex = 0;
-	uint8_t* buff = malloc(sizeof(uint8_t) * buffSize);
+    parseFunctionParameters((AST_FUNCTION_DEC*)func, result, index + 3);
 
-	for(; index <= result.size; ++index) {
-		TOKEN t = result.tokens[index];
+    index = func->endingIndex;
```

cppcheck (2.10-2)

[error] Uninitialized struct member

(uninitStructMember)